tools/setup/optimize-svg

```diff
@@ -42,7 +42,7 @@ if [ "$#" -gt 0 ]; then
 fi
 
 ZULIP_PATH="$(readlink -f "$(dirname "$0")"/../..)"
-YARN="/srv/zulip-yarn/bin/yarn"
+YARN="/usr/local/bin/yarn"
```

Comment:
Hmm, if we're using a hardcoded path of this form, in what sense are we supporting multiple Yarn installations?

Reply:
Corepack manages `/usr/local/bin/yarn` as a symlink to its own code that selects the right version dynamically. See the documentation I linked: https://nodejs.org/api/corepack.html

Reply:
Oh, is the mechanism that `/usr/local/bin/yarn` is a fancy thing that looks at the `package.json` and uses that to determine which version of actual `yarn` to potentially download and use? OK.

Then the other concern I have is whether we need to do something to tell `docker-zulip` to actually download real Yarn so that it'll work offline ... I guess not because we only use `yarn` during the image build process anyway.

Reply:
That, and also Yarn can’t do anything with `--prefer-offline` unless it’s previously been successfully invoked online to populate its cache.